src/Qir/Runtime/lib/Tracer/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ target_link_libraries(Microsoft.Quantum.Qir.Tracer
   "-L${CMAKE_BINARY_DIR}/lib/QIR"
   -lMicrosoft.Quantum.Qir.Runtime
 )
-add_dependencies(Microsoft.Quantum.Qir.QSharp.Foundation Microsoft.Quantum.Qir.Runtime)
+add_dependencies(Microsoft.Quantum.Qir.Tracer Microsoft.Quantum.Qir.Runtime)

```

Comment:
This is just a minor bug fix. The first argument to `add_dependencies` is supposed to be the name of the target that you want to declare dependencies for. In this case, copy-paste mistake had the wrong name here, so it was possible for the tracer to sometimes build before the runtime, resulting in a failure to find the dependent dll. This was causing intermittent failures in the QIR Runtime build in the pipeline.